lib/mongodb.js

```diff
@@ -1038,7 +1038,7 @@ MongoDB.prototype.all = function all(model, filter, options, callback) {
     if (err) {
       return callback(err);
     }
-    var order = {};
+    var order = self.buildSort(model, filter.order);
```

Comment:
buildSort function and Line#1045-1050 contain same logic:
```js
if (!order) {
    var idNames = this.idNames(model);
    if (idNames && idNames.length) {
      order = idNames;
    }
  }
```
does it have to be in both place?